src/core/Track.cpp

```diff
@@ -698,52 +717,33 @@ void TrackContentObjectView::mousePressEvent( QMouseEvent * me )
 			}
 			else
 			{
-				gui->songEditor()->m_editor->selectAllTcos( false );
-				QVector<TrackContentObjectView *> tcoViews;
-				tcoViews.push_back( this );
-				DataFile dataFile = createTCODataFiles( tcoViews );
-				QPixmap thumbnail = QPixmap::grabWidget( this ).scaled(
-							128, 128,
-							Qt::KeepAspectRatio,
-							Qt::SmoothTransformation );
-				new StringPairDrag( QString( "tco_%1" ).arg(
-										m_tco->getTrack()->type() ),
-									dataFile.toString(), thumbnail, this );
+				m_action = ToggleSelected;
```

Comment:
@BaraMGB  I guess something went wrong when you sync with `master` branch.
See https://github.com/LMMS/lmms/pull/3649/files#diff-cb2a453c979a4777b9330e2b96644f0eR701.